admin-manual/installation-setup/customization/customization.rst

```diff
@@ -119,10 +120,24 @@ users to configure their MCP clients in order to control whether or not these
 output streams are captured. See :ref:`Task output capturing configuration
 <task-output-capturing-admin>` for more details.
 
+.. _customize-scalability:
+
+Customizing for scalability and performance
+-------------------------------------------
+
+Archivematica includes a number of features and options to improve scalability
+& performance. The default configuration options are geared towards “typical” 
```

Comment:
We may want to replace `”` with `"`? This probably origins from using a rich editor at some point.

Reply:
ah, yes, probably because I started my draft in google docs? There will be several of those... I will find them all and correct.